pkgs/development/python-modules/pyfcm/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, requests-toolbelt
+}:
+
+buildPythonPackage rec {
+  pname = "pyfcm";
+  version = "1.4.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "cce6c999d8ffe39a44c42329f40c73306586102c6379fc726ea217070b72b967";
+  };
+
+  propagatedBuildInputs = [ requests-toolbelt ];
+
+  # Pypi does not provide test files
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Python client for FCM - Firebase Cloud Messaging (Android, iOS and Web)";
```

Comment:
would you be willing to add yourself as a maintainer?

Reply:
What would this imply? just being on top of new releases to update the package's version?

Reply:
@ldelelis yes, and also helping with issues if they arise. Join us, it's fun.